src/Algebra/Graph.hs

```diff
@@ -859,7 +871,7 @@ filterContext s i o g = maybe g go $ context (==s) g
 -- @
 {-# SPECIALISE replaceVertex :: Int -> Int -> Graph Int -> Graph Int #-}
 replaceVertex :: Eq a => a -> a -> Graph a -> Graph a
-replaceVertex u v = fmap $ \w -> if w == u then v else w
+replaceVertex u v = mapG $ \w -> if w == u then v else w
```

Comment:
Can you keep `fmap` here? It will be rewritten to `mapG` anyway. I think `fmap` is easier to understand.